content/id/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -0,0 +1,121 @@
+                    </ol>
+                      <div class="carousel-inner" role="listbox">
+                        <div class="item carousel-item active">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling1.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling2.svg">
+                        </div>
+                      </div>
+
+                      <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                        <span class="sr-only ">Sebelumnya</span>
+                      </a>
+                      <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                        <span class="sr-only">Selanjutnya</span>
+                      </a>
+
+                    </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Perluasan skala Deployment akan memastikan Pod baru dibuat dan dijadwalkan ke Node-Node dengan sumber daya yang tersedia. Penyekalaan akan meningkatkan jumlah Pod ke keadaan yang diinginkan. Kubernetes juga mendukung <a href="/id/docs/tasks/run-application/horizontal-pod-autoscale/"><i>autoscaling</i></a> Pod, tetapi itu di luar cakupan tutorial ini. Penyekalaan ke nol juga dimungkinkan, dan tindakan ini akan mengakhiri semua Pod pada Deployment tersebut.</p>
+
+                <p>Menjalankan beberapa instans dari aplikasi akan membutuhkan cara untuk mendistribusikan kunjungan ke semuanya. Service punya penyeimbang beban terintegrasi yang akan mendistribusikan kujungan jaringan ke semua Pod dari sebuah Deployment yang diekspos. Service akan terus-menerus memonitor Pod-Pod yang berjalan menggunakan Endpoints, untuk memastikan kunjungan hanya dikirim kepada Pod yang tersedia.</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Penyekalaan dapat dicapai dengan mengubah jumlah replicas pada sebuah Deployment.</i></p>
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>Sewaktu kamu punya multipel instans dari aplikasi berjalan, kamu akan dapat melakukan pembaruan bertahap tanpa perhentian. Kita akan akan membahas hal tersebut pada modul selanjutnya. Sekarang, mari kita pergi ke terminal daring dan menyekalakan aplikasi kita.</p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button"> Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```

Comment:
Link to ID doc, if available.
```suggestion
                <a class="btn btn-lg btn-success" href="/id/docs/tutorials/kubernetes-basics/scale/scale-interactive/" role="button"> Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```